critiquebrainz/db/users.py

```diff
@@ -0,0 +1,51 @@
+from hashlib import md5
+import sqlalchemy
+from critiquebrainz import db
+
+
+def avatar(user):
+    """Args:
+        user: user data (dict or str)
```

Comment:
Because it was meant to make a universal function for db, so there is no need to write another one in future. I don't see anything bad in this implementation.